content/docs/studio/user-guide/connect-custom-gitlab-server.md

```diff
@@ -49,3 +49,20 @@ Custom GitLab server support is available only within a team workspace, and is
 available to all Iterative Studio teams (free and paid).
 
 </admon>
+
+## My GitLab server is behind a firewall. What should I do?
```

Comment:
Can be also useful / was asked for for for the on-prem storage (SSH).

Reply:
Oh, I didn't know that. Let me add this somewhere ;-)

Reply:
@shcheklein can you re-check again?